### ISSUE TITLE
Is the code coming？

### ISSUE THREAD

**Issue by AITech-D** [LukeCho-8810/HITNet-Pytorch#1]
Thank you for your wonderful work.  I've read your paper. It's very novel and interesting.   When to upload your wonderful code?  I can't wait to read it.

**Comment by sohils** [comment]
Hi @LukeCho-8810, I would love to know if you are working on implementing this code?

**Comment by AITech-D** [comment]
> Hi @LukeCho-8810, I would love to know if you are working on implementing this code?
Yes, But I have some problems， I would love to refer to your code。So，When will you release the code？Thank U.

**Comment by yellowYuga** [comment]
> > Hi @LukeCho-8810, I would love to know if you are working on implementing this code?
> > Yes, But I have some problems， I would love to refer to your code。So，When will you release the code？Thank U.

Hi, @LukeCho-8810 ,I'm working on this paper too.But I found some problems in my experiments.I don't see any email address or other contact information of yours. If you or anyone want to discuss please contact me with no hesitation.

**Comment by meteorshowers** [comment]
interesting hhh

**Comment by MJITG** [comment]
Hey, are you guys still working on this implementation? If so, why don't we collaborate to work it out? I'm going to reproduce this paper.

**Comment by MJITG** [comment]
>Hey, are you guys still working on this implementation? If so, why don't we collaborate to work it out? I'm going to reproduce this paper.
 @AITech-D @sohils @yellowYuga @LukeCho-8810 @meteorshowers

**Comment by MJITG** [comment]
Hey guys, I've uploaded my implementation to my repository. Have a check [here](https://github.com/MJITG/PyTorch-HITNet-Hierarchical-Iterative-Tile-Refinement-Network-for-Real-time-Stereo-Matching):)
